utils/locale/src/locale.mjs

```diff
@@ -0,0 +1,289 @@
+// @todo update this to import it from `@ma-shop/is`
+
+import { find as lodashFind, omit } from 'lodash'
+
+
+const find = (values, _, omittedValues = []) => {
+  let obj = _
+  // if it's a string assume it's by values
+  if (typeof obj === 'string') {
+    obj = (item) => Object.values(omit(item, ...omittedValues)).includes(_)
+  }
+  return lodashFind(values, obj)
+}
+
+export class LocaleManager {
+  static localeItems = [
+    {
+      locale: 'en-US',
+      name: 'United States',
+      language: 'en',
+      country: 'US',
+      alpha2: 'US',
+      alpha3: 'USA',
+      curreny: 'USD',
+      ///# @note https://github.com/moment/moment/issues/3624
+      moment: 'en',
+      isMarketCountry: true,
+    },
+    {
+      locale: 'en-AU',
+      name: 'Australia',
+      language: 'en',
+      country: 'AU',
+      alpha2: 'AU',
+      alpha3: 'AUS',
+      curreny: 'AUD',
+      moment: 'en-au',
+      isMarketCountry: true,
+    },
+    {
+      locale: 'en-CA',
+      name: 'Canada',
+      language: 'en',
+      country: 'CA',
+      alpha2: 'CA',
+      alpha3: 'CAN',
+      curreny: 'CAD',
+      moment: 'en-ca',
+      isMarketCountry: true,
+    },
+    {
+      locale: 'fr-CA',
+      language: 'fr',
+      name: 'Canada',
+      country: 'CA',
+      alpha2: 'CA',
+      alpha3: 'CAN',
+      curreny: 'CAD',
+      moment: 'fr-ca',
+      isMarketCountry: false,
```

Comment:
This shouldn't be false unless `fr-CA` is meant to only specify quebec or another french region